### DISCUSSION TITLE
mc:Setting policy for an user

### DISCUSSION THREAD

**Discussion by spbho** [minio/minio#13563] [Q&A]
I am able to set alias for hostname with default username and password given in helm chart but for setting policy getting below issue
./mc alias set minio2 http://hostname:port admin admin(host is accessible when browser is set with Ingresscontroller ip and port)
Added minio2 successfully.
./mc admin policy set minio2 readwrite user='uid=,ou=,dc=,dc=' (same lookup bind dn from LDAP)
mc: Unable to set the policy. Failed to parse server response: invalid character 'd' looking for beginning of value.
I want to understand if this is correct approach or not to set policy for an LDAP user
I have gone through mc and LDAP documentation, but not able to get the solution

**Comment by vadmeste** [comment]
> mc: Unable to set the policy. Failed to parse server response: invalid character 'd' looking for beginning of value.

Most likely the endpoint is wrong.

What's the output of `curl http://hostname:port` here ?

**Reply by spbho** [reply]
> > mc: Unable to set the policy. Failed to parse server response: invalid character 'd' looking for beginning of value.
> 
> Most likely the endpoint is wrong.
> 
> What's the output of `curl http://hostname:port` here ?

default backend - 404
As in browser I am accessing this http://hostname by setting proxy with Ingresscontroller ip and port
How to apply policy in this case?

**Comment by spbho** [comment]
Instead of ingress hostname, I am trying to set it for LoadBalancer service but getting issue in alias itself but able to set alias for hostname, what's the difference
./mc alias set minio3 http://ipaddress:9000 admin admin
Console is accessible with http://ipaddress:9000 ldap username and password same as hostname-http://hostname ldap username and password with proxy 
mc: <ERROR> Unable to initialize new alias from the provided credentials. XML syntax error on line 1: element <link> closed by </h                                   ead>.
API Port-9000
Console port-9001

**Reply by vadmeste** [reply]
@spbho you simply have issues configuration ingress/loadbalancer in your case. You need to figure this is out. This is how to test you are talking with MinIO:
```
$ curl https://play.min.io:9000
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Resource>/</Resource><Region>us-east-1</Region><RequestId>16B3FF93F11BA216</RequestId><HostId>c1758167-4426-454f-9aae-5c3dfdf6df64</HostId></Error>
```
